File: 1048/1048.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var sal = parseFloat(lines.shift());
var bonus;

if (sal <= 400) {
  bonus = 0.15;
} else if (sal <= 800) {
  bonus = 0.12;
} else if (sal <= 1200) {
  bonus = 0.1;
} else if (sal <= 2000) {
  bonus = 0.07;
} else {
  bonus = 0.04;
}

var reajuste = sal * bonus;
var novo = sal + reajuste;
var percent = parseInt(bonus * 100);

console.log("Novo salario: " + novo.toFixed(2));
console.log("Reajuste ganho: " + reajuste.toFixed(2));
console.log("Em percentual: " + percent + " %");

File: 1048/1048.py
sal = float(input())

if sal <= 400:
    bonus = 0.15
elif sal <= 800:
    bonus = 0.12
elif sal <= 1200:
    bonus = 0.10
elif sal <= 2000:
    bonus = 0.07
else:
    bonus = 0.04

reajuste = sal * bonus
novo = sal + reajuste
percent = int(bonus * 100)

print(f"Novo salario: {novo:.2f}")
print(f"Reajuste ganho: {reajuste:.2f}")
print(f"Em percentual: {percent} %")

